Feature: add GitHub contribution links to footer

AngelCodes95 opened the pull request:
Summary

  Added GitHub contribution links to the landing page footer to improve community engagement and project accessibility.

  Changes

  - Footer Enhancement: Added two contribution links in the footer section
    - Link to landing page repository contribution guide
    - Link to Terminal Jarvis CLI contribution documentation
  - Contribution Documentation: Created CONTRIBUTING.md with development guidelines, setup instructions, and community standards in root of project
  - Styling Implementation: Added CSS theming for contribution links with hover effects and CLI-specific glow styling
  - Package Update: Minor version bump to reflect new community features per https://semver.org/

  Technical Details
  
  - Special glow effect for CLI contribution link using text-shadow
  - Both links open in new tabs with proper security attributes
  - Terminal monospace font styling maintains design consistency

  Test Plan Overview

  - Verified both contribution links navigate to correct repositories
  - Tested hover effects and styling in both light/dark themes
  - Confirm responsive layout on mobile and desktop
  - Validate new tab opening behavior

Sneak Peak

<img width="721" height="246" alt="tjl-contribute" src="https://github.com/user-attachments/assets/894b53e7-a172-41ce-8aad-ad2e8c5e01ab" />
<img width="714" height="239" alt="tjl-light-contribute" src="https://github.com/user-attachments/assets/d2cc4228-0c63-459e-a0c0-7e9e98226d87" />

AngelCodes95 committed:
Feature: add GitHub contribution links to footer

Add contribution links for landing page and CLI repositories
Create CONTRIBUTING.md with development guidelines
Implement CSS theming for link styling

AngelCodes95 commented:
this PR addresses issue #3

BA-CalderonMorales commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,213 @@
+
+1. **Make sure you are up to date on branch**
+   ```bash
+   git checkout development
````
to stay consistent with the tj repo, let’s use the following branches as core branches:

- develop
- main

this suggestion is out of scope for this change, just noticed it was different.

AngelCodes95 replied:
I agree, can tackle that as another issue.

BA-CalderonMorales commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,213 @@
+
+```bash
+# Examples:
+git commit -m "Fix: resolve mobile responsive layout issue"
````
lets keep consistency:

- feat:
- docs:
- chore:
- refactor:
- style:

instead of initial letter uppercase.

this helps keeps commits following a standard.

AngelCodes95 replied:
simple enough, works for me!

BA-CalderonMorales commented on CONTRIBUTING.md:
````diff
@@ -0,0 +1,213 @@
+```
+
+**Semantic Versioning**
+- `Feature:` minor version bump (1.0.0 → 1.1.0)
````
if you do follow my other suggestion, we’ll need to keep it jn mind for when we start looking into implementing semantic versionjng via gh actions.

BA-CalderonMorales reviewed (commented):
nice! just make sure to get some more feedback from team on wording.

BA-CalderonMorales commented:
the wording I am talking about is:

Contribute to [x] | Contribute to [y]

Could be phrased differently. If you don’t get feedback by EOD today though, go ahead and ship.

AngelCodes95 commented:
https://github.com/user-attachments/assets/c1d59959-7801-4d2d-93f1-90fa787b89f7

latest update

AngelCodes95 committed:
feat: address PR feedback - enhance contribution links

- Add GitHub icon and terminal cursor
- Update styling and remove glow effects
- Clean up unused CSS variables
